jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/OtelHelper.groovy

```diff
@@ -86,24 +86,40 @@ class OtelHelper {
      * Returns an updated @{link InstrumentHelper} associated with the provided {@link MBeanHelper} and its specified
      * attribute value(s).  The parameters map to the InstrumentHelper constructor.
      */
-    InstrumentHelper instrument(MBeanHelper mBeanHelper, String instrumentName, String description, String unit, Map<String, Closure> labelFuncs, String attribute, Closure otelInstrument) {
-        def instrumentHelper = new InstrumentHelper(mBeanHelper, instrumentName, description, unit, labelFuncs, attribute, otelInstrument)
+    InstrumentHelper instrument(MBeanHelper mBeanHelper, String instrumentName, String description, String unit, Map<String, Closure> labelFuncs, Map<String, Map<String, Closure>> attributes, Closure otelInstrument) {
+        def instrumentHelper = new InstrumentHelper(mBeanHelper, instrumentName, description, unit, labelFuncs, attributes, otelInstrument)
         instrumentHelper.update()
         return instrumentHelper
     }
 
+    InstrumentHelper instrument(MBeanHelper mBeanHelper, String instrumentName, String description, String unit, Map<String, Closure> labelFuncs, String attribute, Closure otelInstrument) {
+      instrument(mBeanHelper, instrumentName, description, unit, labelFuncs, [(attribute): null], otelInstrument)
```

Comment:
Since groovy was updated I guess explicit coercion is no longer required here but I wonder if this is preferable to using the NullObject:

```suggestion
      instrument(mBeanHelper, instrumentName, description, unit, labelFuncs, [(attribute): [:] as Map<String, Closure>], otelInstrument)
```